src/engraving/libmscore/rest.cpp

```diff
@@ -334,6 +343,13 @@ void Rest::layout()
     for (EngravingItem* e : el()) {
         e->layout();
     }
+
+    _skipDraw = false;
+    if (_deadSlapped) {
+        _skipDraw = true;
+        return;
+    }
```

Comment:
Yeah, this seems a bit hacky to me (see also my previous comment). If I understand correctly, we are just drawing the dead slapped instead of the rest. I think it can do for now, but I'd consider switching to a more solid solution in future